helpers/ifrauBackdropService.js

```diff
@@ -1,10 +1,13 @@
+import { isFramed } from './framed.js';
+import { SlimClient } from 'ifrau/client/slim.js';
+
 let ifrauBackdropService;
 
 export async function tryGetIfrauBackdropService() {
-	if (!window.ifrauclient) return null;
+	if (!await isFramed()) return null;
 	if (ifrauBackdropService) return ifrauBackdropService;
 
-	const ifrauClient = await window.ifrauclient().connect();
+	const ifrauClient = await new SlimClient().connect();
```

Comment:
Would a dynamic import of the slim client work here? That'd avoid making the HTTP request until it's actually needed instead of always.

Reply:
I imagine it certainly would, yes. I can change this.